samtranslator/intrinsics/actions.py

```diff
@@ -5,6 +5,29 @@
 from samtranslator.model.exceptions import InvalidDocumentException, InvalidTemplateException
 
 
+def _get_parameter_value(parameters: Dict[str, Any], param_name: str, default: Any = None) -> Any:
+    """
+    Get parameter value from parameters dict, but return default (None) if it's a CloudFormation internal placeholder.
```

Comment:
Minor, but the comment should also mention that it returns the default when the `param_name` is not in the `parameters`. I think in that case the CloudFormation deployment might fail overall, but at least in this function context that case is supposed to be handled too..